version_control/katalyst_by_codurance/templates/home.html

```diff
@@ -26,6 +26,10 @@ <h1>{% module "hero_title" label="Hero title" path="@hubspot/text" no_wrapper=Tr
 
 {# 2 INTRO TEXT + ICON WITH TITLE AND SUBTITLE #}
 {# 3 CLIENTS LOGO CLOUD #}
+<section class="client-logos container">
+    {% module "clients_section_header" path="@hubspot/section_header" label="Clients logos header"%}
```

Comment:
Personally I wouldn't use the section_header module here, since in this section there's only going to be a single title. Maybe we could use a simple text tag or similar